spec/factories/lettings_log.rb

```diff
@@ -190,6 +190,9 @@
       status { 4 }
       discarded_at { Time.zone.now }
     end
+    trait :imported do
+      old_id { rand(1...99_999) }
```

Comment:
isn't this the format of the old_form_ids?
I think the old_ids are long hex strings

Reply:
Sure, updated now!